tasks/templates/tasks/view_single_task.html

```diff
@@ -1,47 +1,63 @@
 {% extends "base.html" %}
 {% block content %}
-    {% if user.user.is_authenticated %}
-        <h2>{{ task.title }}</h2>
-        <article class="media content-section">
-            <div class="media_body">
-                <h6><u>Description:</u></h6>
-                <p class="article-content">{{ task.description }}</p>
-                <div class="article_metadata">
-                    <small class="text-muted">
-                        <b>Created on: </b>{{ task.created_date|date:"F d, Y" }} |
-                        <b>Created by: </b>{{ task.created_by.user.first_name }} {{ task.created_by.user.last_name }} |
-                        <b>Assignee: </b>{{ task.assignee.user.first_name }} {{ task.assignee.user.last_name }} |
-                        <b>Priority: </b>{{ task.priority }} |
-                        <b>Task Id: </b>{{ task.id }}
-                    </small>
-                </div>
-            </div>
-            {% if user.is_employee %}
-                <form action="" method="POST">
-                {% csrf_token %}
-                <b>{{ form }}</b>
-                <input class="btn btn-secondary btn-sm mt-1" type="submit" value="Change Status"/>
-            {% endif %}
-            {% if user.is_manager %}
-                <div>
-                    <a class="btn btn-secondary btn-sm mt-1 md-1" href="edit/{{ task.id }}">Edit</a>
-                    <a class="btn btn-danger btn-sm mt-1 md-1">Delete</a>
-                </div>
-            {% endif %}
-        </article>
-        <h3>Comments</h3>
-        {% for comment in task.get_comments %}
-            <article class="media content-section">
-                <div class="media_body">
-                    <div class="article_metadata">
-                        <b>{{ comment.appUser.user.first_name|truncatechars:20 }} {{ comment.appUser.user.last_name|truncatechars:20 }}</b>
-                        <small class="text-muted">
-                            {{ comment.created_date|date:"F d, Y" }}
-                        </small>
-                    </div>
-                    <p class="article-content">{{ comment.description|truncatechars:400 }}</p>
-                </div>
-            </article>
+{% if user.user.is_authenticated %}
+<h2>{{ task.title }}</h2>
```

Comment:
since the "{% block content %}" is also block of code with close command "{% endblock content %}"
I would suggest to add right indentation to the line below it, in this way the code is more readable.
```suggestion
    {% if user.user.is_authenticated %}
        <h2>{{ task.title }}</h2>
```